tests/aggregationsSpec.js

```diff
@@ -22,6 +22,142 @@ describe('aggregations', function() {
     actors: ['e']
   }]
 
+  var moreItems = [{
```

Comment:
That json array should go to './fixtures/movies_some_name.json' and be loaded here like `var moreItems = require('./fixtures/movies_some_name.json')`